src/paper/related_models/paper_model.py

```diff
@@ -59,8 +59,22 @@
 HELP_TEXT_IS_PDF_REMOVED = "Hides the PDF because it infringes Copyright."
 
 
-class Paper(AbstractGenericReactionModel):
-    history = HistoricalRecords()
+class AbstractPaper(AbstractGenericReactionModel):
```

Comment:
This abstract class was required to make the methods available on the HistoricalPaper class